posts/2021/10/interchange-dataframe-protocol-for-cudf.md

```diff
@@ -0,0 +1,351 @@
+<!--
+.. title: Interchange dataframe protocol: cuDF implementation
+.. slug: data-apis-cudf
+.. date: 2021-10-05 16:10:00 UTC+00:00
```

Comment:
```suggestion
.. date: 2021-10-21 09:00:00 UTC+00:00
```
@iskode @melissawm i've modified the publish date for this thursday. are there any major problems that would hinder this from publishing on this coming thursday?